rebasehelper/spec_hooks/pypi_url_fix.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you to rebase package to the latest version
+# Copyright (C) 2013-2014 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# he Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hracek <phracek@redhat.com>
+#          Tomas Hozza <thozza@redhat.com>
+
+import re
+from rebasehelper.logger import logger
+
+from rebasehelper.specfile import BaseSpecHook
+
+
+class PypiUrlFixHook(BaseSpecHook):
+    """SpecHook for transforming old nonfunctional python package url to
+    a safe url(eg. https://pypi.* to https://files.pythonhosted.*)"""
```

Comment:
[Multi-line docstring](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings) should be formatted differently.